src/skin/firstRun.html

```diff
@@ -1,73 +1,186 @@
-  -
-  - Adblock Plus is free software: you can redistribute it and/or modify
-  - it under the terms of the GNU General Public License version 3 as
-  - published by the Free Software Foundation.
-  -
-  - Adblock Plus is distributed in the hope that it will be useful,
-  - but WITHOUT ANY WARRANTY; without even the implied warranty of
-  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-  - GNU General Public License for more details.
-  -
-  - You should have received a copy of the GNU General Public License
-  - along with Adblock Plus.  If not, see <http://www.gnu.org/licenses/>.
-  --> <html> <head>
-    <title class="i18n_firstRun_title"></title>
-    <meta charset="utf-8">
-    <script type="text/javascript" src="/lib/i18n.js" charset="utf-8"></script>
-    <script type="text/javascript" src="/lib/vendor/jquery-3.3.1.min.js"></script>
-    <script type="text/javascript" src="/lib/vendor/jquery-ui-1.12.1.custom/jquery-ui.min.js"></script>
-    <script type="text/javascript" src="/lib/jquery.jcarousel-core.min.js"></script>
-    <script type="text/javascript" src="/skin/firstRun.js"></script>
-    <link type="text/css" href="/lib/vendor/jquery-ui-1.12.1.custom/jquery-ui.structure.min.css" rel="stylesheet" />
-    <link type="text/css" href="/lib/vendor/jquery-ui-1.12.1.custom/jquery-ui.theme.min.css" rel="stylesheet" />
-    <link type="text/css" href="/skin/firstRun.css" rel="stylesheet" />
-    <style>
-      </style>
-  </head>
-  <body>
-      <!--p class="i18n_first_run_text"></p -->
-    <div id="header-container">
-      <div id='eff-header-wrapper'><div id="eff-header"><a href="https://www.eff.org"><img src="/skin/eff-logo.png" width=45 height=32><span class="i18n_intro_by_eff eff-text"></span></a></div></div>
-      <div id="badger-header-wrapper"><div id='badger-header'><img id="header-banner-img" src="/icons/PB-header-1.png"></div> </div>
+- This file is part of Adblock Plus <http://adblockplus.org/>,
+- Copyright (C) 2006-2013 Eyeo GmbH
+-
+- Adblock Plus is free software: you can redistribute it and/or modify
+- it under the terms of the GNU General Public License version 3 as
+- published by the Free Software Foundation.
+-
+- Adblock Plus is distributed in the hope that it will be useful,
+- but WITHOUT ANY WARRANTY; without even the implied warranty of
+- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+- GNU General Public License for more details.
+-
+- You should have received a copy of the GNU General Public License
+- along with Adblock Plus.  If not, see <http://www.gnu.org/licenses/>.
+
+-->
+<!doctype html>
+<html class="no-js" lang="en">
+<head>
+  <title class="i18n_firstRun_title"></title>
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link type="text/css" href="css/pb.css" rel="stylesheet" />
+  <script type="text/javascript" src="js/jquery.min.js" ></script>
```

Comment:
Could you reuse the jQuery we use elsewhere (in `src/lib/vendor`)?

Reply:
Yes - to using CSS directly. It's much, much easier for me to code with sass + foundation, but everything is compiled to pb.css for you to use.
Yes to using existing jquery paths.